### PR TITLE
Add azure pipeline

### PR DESCRIPTION
**Pull request by DonJayamanne**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-    "name": "@vscode/jupyter-ipywidgets",
-    "version": "1.0.9",
+    "name": "@vscode/jupyter-ipywidgets7",
```
Comment (DonJayamanne) [inline package.json:2]: Change name to suffix with `7`, as the new version will be supporting v8 of IPYWidgets and that will not have a version suffix.
This allows us to have the two npm pacakges installed in Jupyter extension

File: package.json [hunk 2]
```diff
@@ -55,6 +56,7 @@
         "prettier": "^2.6.2",
         "rimraf": "^3.0.2",
         "style-loader": "^3.3.1",
+        "typescript": "4.7",
```
Comment (DonJayamanne) [inline package.json:59]: For some reason this wasn't added as a dependency, 
Even though 4.7.3 it was in package-lock.json (not sure what happened or how the package-lock was generated).
this change ensures we have 4.7.4 defined in both files.

File: webpack.config.js [hunk 3]
```diff
@@ -18,6 +18,7 @@ module.exports = [
         devtool: isProdBuild ? 'source-map' : 'inline-source-map',
         entry: path.join(outDir, 'index.js'),
         cache: true,
+        externals: ['crypto'],
```
Comment (DonJayamanne) [inline webpack.config.js:21]: We will not ship crypto, required by @lumino/coreutils

**Comment by lszomoru** [comment]
@DonJayamanne, I a little confused by the fact that you PR pipeline is extending the "extension" template while the other pipeline is extending the "nom package" pipeline. This makes it very difficult for me to understand what we are trying to achieve here.

**Comment by DonJayamanne** [comment]
> PR pipeline is extending the "extension" template while the other pipeline is extending the "nom package" pipeline.

Aah sorry, was hoping to use an existing template for PRs.
Didn't realize that was specifically created for extensions
Will remove that.

Can I use the `npm package` pipeline for PRs as well.
I.e. use that pipeline to validate PRs (without the publish stage).


Again, will remove that other PR pipeline.
& will merge this PR & remove the two pipelines I created (as per your request in email) so you can create the pipeline correctly.

**Pull request merged**